programs/client/CMakeLists.txt

```diff
@@ -1,6 +1,22 @@
+PROTOBUF_GENERATE_CPP(GRAMMAR_SRC GRAMMAR_HDR buzz-house/ast/sql_grammar.proto)
+
 set (CLICKHOUSE_CLIENT_SOURCES
     Client.cpp
+    # BuzzHouse
+    buzz-house/ast/sql_query_proto_to_string.cpp
+    buzz-house/generator/sql_query.cpp
+    buzz-house/generator/sql_expression.cpp
+    buzz-house/generator/sql_table.cpp
+    buzz-house/generator/sql_types.cpp
+    buzz-house/generator/statement_generator.cpp
+    buzz-house/generator/random_settings.cpp
+    buzz-house/generator/query_oracle.cpp
+    buzz-house/generator/external_integrations.cpp
+    buzz-house/third_party/hugeint.cpp
+    buzz-house/third_party/uhugeint.cpp
+    ${GRAMMAR_SRC}
 )
+set_source_files_properties(${CLICKHOUSE_CLIENT_SOURCES} PROPERTIES COMPILE_FLAGS "-Wno-reserved-identifier -Wno-extra-semi-stmt -Wno-used-but-marked-unused -Wno-covered-switch-default")
```

Comment:
Why did you have to disable all these warnings for buzz-house?

If it's for our own buzz-house code, I think we should stick to the same standards as the rest of the code.
If it's third-party code, we can use pragma statements to change diagnostics of just those files, or change the compiler flags only for those files.

Reply:
Removed a few of them. These flags are needed because of Protobuf. Codegen fuzzer also sets them.